systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectST.java

```diff
@@ -303,19 +303,15 @@ void testKafkaConnectWithPlainAndScramShaAuthentication() {
         KafkaConnectorUtils.createFileSinkConnector(testStorage.getNamespaceName(), scraperPodName, testStorage.getTopicName(),
             TestConstants.DEFAULT_SINK_FILE_PATH, KafkaConnectResources.url(testStorage.getClusterName(), testStorage.getNamespaceName(), 8083));
 
-        KafkaClients kafkaClients = new KafkaClientsBuilder()
-            .withTopicName(testStorage.getTopicName())
-            .withMessageCount(testStorage.getMessageCount())
+        // kafka sink connect will expect 100 messages
+        KafkaClients kafkaClients = ClientUtils.getInstantPlainClientBuilder(testStorage)
             .withUsername(testStorage.getUsername())
-            .withBootstrapAddress(KafkaResources.plainBootstrapAddress(testStorage.getClusterName()))
-            .withProducerName(testStorage.getProducerName())
-            .withConsumerName(testStorage.getConsumerName())
-            .withNamespaceName(testStorage.getNamespaceName())
+            .withMessageCount(100)
```

Comment:
the `getInstantPlainClientBuilder` already contains this message count, right?

Reply:
there and in others point you are mentioning this is some actual "dependency" on number 100 to be send. In this case, but i guess this could be a bit refactor to get rid of this completely.